### PR TITLE
Add `WAIT` examples to START/STOP sections and syntax tables to the Cypher adm commands (#1615)

### PR DESCRIPTION
**Pull request by NataliaIvakina**
- Syntax tables were added to the modified pages
- Pages were restructured in line with the neo4j-admin commands section
- Usage examples were aligned

---------

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1654.surge.sh

Updated pages:
- [Alter databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/alter-databases)
- [Configuration parameters](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/configuration-parameters)
- [Create, start, and stop databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Delete databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [List databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/listing-databases)
- [<code>WAIT</code> options](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/wait-options)

File: modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc [hunk 1]
```diff
@@ -149,25 +119,131 @@ More details about seeding options can be found in xref::clustering/databases.ad
...
+|===
+| Command | Syntax
+
+| START DATABASE
+|
+[source, syntax, role="noheader"]
+----
+START DATABASE name [WAIT [n [SEC[OND[S]]]]\|NOWAIT]
+----
+
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/database-administration/standard-databases/create-databases.adoc:229]:
```suggestion
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1654.surge.sh

Updated pages:
- [Manage roles](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/authentication-authorization/manage-roles)
- [Alter databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/alter-databases)
- [Configuration parameters](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/configuration-parameters)
- [Create, start, and stop databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Delete databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [List databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/listing-databases)
- [<code>WAIT</code> options](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/wait-options)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1654.surge.sh

Updated pages:
- [Manage roles](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/authentication-authorization/manage-roles)
- [Alter databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/alter-databases)
- [Configuration parameters](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/configuration-parameters)
- [Create, start, and stop databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Delete databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [List databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/listing-databases)
- [<code>WAIT</code> options](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/wait-options)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1654.surge.sh

Updated pages:
- [Manage roles](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/authentication-authorization/manage-roles)
- [Alter databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/alter-databases)
- [Configuration parameters](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/configuration-parameters)
- [Create, start, and stop databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Delete databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [List databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/listing-databases)
- [<code>WAIT</code> options](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/wait-options)
- [Administering immutable privileges](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/tutorial/tutorial-immutable-privileges)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1654.surge.sh

Updated pages:
- [Manage roles](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/authentication-authorization/manage-roles)
- [Alter databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/alter-databases)
- [Configuration parameters](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/configuration-parameters)
- [Create, start, and stop databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Delete databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [List databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/listing-databases)
- [<code>WAIT</code> options](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/wait-options)
- [Administering immutable privileges](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/tutorial/tutorial-immutable-privileges)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1654.surge.sh

Updated pages:
- [Manage roles](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/authentication-authorization/manage-roles)
- [Alter databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/alter-databases)
- [Configuration parameters](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/configuration-parameters)
- [Create, start, and stop databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/create-databases)
- [Delete databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/delete-databases)
- [List databases](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/listing-databases)
- [<code>WAIT</code> options](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/database-administration/standard-databases/wait-options)
- [Administering immutable privileges](https://neo4j-docs-operations-1654.surge.sh/operations-manual/5/tutorial/tutorial-immutable-privileges)

**Pull request merged**